about-us.ftd

```diff
@@ -21,7 +21,7 @@ AdMint happened out of curiosity. Pure & simple.
 A mutual interest in the weird, wonderful world of web 3.0, & a penchant for
 AdTech is what got Jay & Harish connect with each other in a hackathon. A few
 months & lot of experiments on, they felt it was time they consolidate all
-their findings into a product. That really was the beginning.
```

Comment:
@SuryanshShahi dont push about-us.ftd as it's removed from `final-stuff` branch.
Take latest pull of `final-stuff` branch and then raise PR.